sdk/docs/manually-written/overview/explanations/ledger-model/ledger-privacy.rst

```diff
@@ -376,3 +373,75 @@ A Create event with `N` observers appears in the projection of at least those `N
+then Bank 2 can infer some information about the *shape* of the original transaction and the parties involved,
+beyond what is visible in the projection.
+This is because the synchronizer needs this shape information for its two-phase commit protocol.
+
+.. important::
+   The contents of the transaction remain confidential even if a party inspects the messages exchanged via the synchronizer.
+   The party can at most reconstruct the shape of the transactions on the ledger.
+
+This section defines the shape that overapproximate the information that can leak via messages exchanged via the synchronizer.
+Overapproximation here means that a curious party may not be able to fully reconstruct the shape of a transaction on the ledger,
+for example due to optimizations in the synchronization protocol.
+
+The synchronization protocol uses a notion of the required confirmers for a node, which are a subset of the informees.
+The required confirmers are therefore part of the shape and defined as follows.
+
+.. admonition:: Definition: required confirmers
+
+   The **required confirmers** for a Create node are the signatories and for an Exercise or Fetch node are the signatories and actors.
+
+The shape of a node is defined as follows:
+   
+.. admonition:: Definition: node shape
+
+   The **shape** of a node consists of the following pieces of information:
+
+   * The informees of the node.
+
+   * The size of the encoding of the node contents and the input contract, if any.
+
+   * The **required confirmers** for the node.
+
+The shape-revealing projection extends the projection defined above with the shape of the omitted nodes.
+
+.. _def-tx-projection-shape:
+
+.. admonition:: Definition: shape-revealing projection
+
+   The **shape-revealing projection of a transaction** for a set `P` of parties is obtained as follows for each root action `act` of the transaction:
+
+   #. If `P` contains at least one of the informees of `act`, keep `act` as-is, including its consequences.
+
+   #. Else, replace the node of `act` with its shape and the shape-revealing projection of the consequences of `act` becomes the children of the node shape.
+
+   The **shape-revealing projection of a ledger** `l` for a set `P` of parties is obtained by replacing the transaction in each commit by its shape-revealing projection for `P` while retaining the update ID and the requesters.
+
+For example, the shape-revealing projection of the :ref:`above DvP ledger <da-dvp-ledger-projections>` for Bank 2 looks as follows.
+Node shapes are shown as empty boxes with a `?` inside and annotated with the informees and the required confirmers (underlined).
+Importantly, the shape-revealing projection retains the requesters and the empty projected transactions, unlike for normal projections.
+Conversely, the node shapes do not show the inputs and outputs.
+This means that parties do not see the transaction graph that is outside their projection.
```

Comment:
```suggestion
Thus despite the shape of individual transactions being visible to all parties, 
the structure of the overall transaction graph remains private. 
```
make this key statement stand out a bit more. my text is just one attempt to make the contrast clear.